site/content/contribute/mobile/e2e/environment-vars.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Environment Variables"
+date: 2020-10-06T09:00:00-00:00
+weight: 7
+subsection: Mobile End-to-End Testing
+---
+
+We use several environment variables for Detox testing in order to:
+- Easily change when running in CI.
+- Cater to different values across developer machines.
+
+Test configurations are [defined at test_config.js](https://github.com/mattermost/mattermost-mobile/blob/master/detox/e2e/support/test_config.js) and environment variables are used to override default values. In most cases you don't need to change the values, because it makes use of the default local developer setup. If you do need to make changes, you may override by exporting, e.g. `export SITE_URL=<site_url>`.
+
+| Variable            | Description                                |
+|---------------------|--------------------------------------------|
+| SITE_URL | Host of test server.<br><br>Default: `http://localhost:8065` for iOS or `http://10.0.2.2:8065` for Android. |
+| ADMIN_USERNAME | Admin's username for the test server.<br><br>Default: `sysadmin` when server is seeded by `make test-data`. |
+| ADMIN_PASSWORD | Admin's password for the test server.<br><br>Default: `Sys@dmin-sample1` when server is seeded by `make test-data`. |
+| LDAP_SERVER | Host of LDAP server.<br><br>Default: `localhost` |
+| LDAP_PORT | Port of LDAP server.<br><br>Default: `389` |
```

Comment:
@saturninoabril will  you be adding the EE_LICENSE variable as well?

Reply:
I'd suggest no at the moment since it's not use to change value programmatically during testing. But I may submit a separate PR to highlight some of the pointers which will include on how to deal with test related to license.
In local dev, main flow is uploading the license via sysadmin UI. In case of license related test (upload/remove), we'll suggest to add `mattermost-license.txt` manually in the expected folder but this is special case.
In CI, it's foreseen to have such environment variable but its not fix and may change depending on how we wanted to handle license in CI.